docs/develop/guides/cheat-sheets/momento-cache-python-cheat-sheet.md

```diff
@@ -0,0 +1,107 @@
+
+## Import libraries and connect to return a CacheClient object
+```python
+from datetime import timedelta
+from momento import CacheClient, Configurations, CredentialProvider
+from momento.responses import CacheGet, CacheSet, CreateCache, ListCaches, CacheIncrement
+
+import os
+
+  def client():
+    momento_auth_token = CredentialProvider.from_environment_variable('MOMENTO_AUTH_TOKEN')
+    ttl  = timedelta(int(os.getenv('MOMENTO_TTL_SECONDS')))
+    config = {
+      'configuration': Configurations.Laptop.v1(),
+      'credential_provider': momento_auth_token,
+      'default_ttl':  ttl
+    }
+    return CacheClient(**config)
+```
+
+## Create a new cache in Momento Serverless Cache
+Use this function to create a new cache in your account
+```python
+  def create_cache(cache_name):
+    with MomentoCounter.client() as client:
+      resp = client.create_cache(cache_name)
+      match resp:
+        case CreateCache.Success():
+            print("Cache created.")
+        case CreateCache.Error() as error:
+            print(f"Error incrementing value: {error.message}")
+        case _ as error:
+            print(f"Unreachable with {error.message}")
+```
+
+## Get list of existing caches in your account
+In this example, we use the client function above to get a client object and then use it to list all of the cache's for this account.
+```python
+  def list_caches() -> None:
+      print("Listing caches:")
+      with MomentoCounter.client() as client:
+        list_caches_response = client.list_caches()
+        match list_caches_response:
+            case ListCaches.Success() as success:
+                for cache_info in success.caches:
+                    print(f"- {cache_info.name!r}")
+            case ListCaches.Error() as error:
+                print(f"Error creating cache: {error.message}")
+            case _:
+                print("Unreachable")
+        print("")
+```
+## Write an item to a cache
+A simple example of doing a set operation. In the client.set call, you could pass a fourth value, the TTL and this would override the default TTL value set with the client connection object.
+```python
+  def set(cache_name, key, value):
+    with MomentoCounter.client() as client:
+      resp = client.set(cache_name,key,value)
+      match resp:
+        case CacheSet.Success():
+            pass
+        case CacheSet.Error() as error:
+            print(f"Error incrementing value: {error.message}")
+        case _:
+            print("Unreachable")
+```
+
+## Read an item from a cache
+This is an example of a simple read operation to get an item from a cache.
+```python
+  def get(cache_name, key):
+    cache_name = os.getenv('MOMENTO_CACHE_NAME')
```

Comment:
We read the cache name from this example function but not in the others. Was this left by mistake?

Reply:
Fixed